### ISSUE TITLE
What is playStoreId and why do I need it to run app on my own device?

### ISSUE THREAD

**Issue by outluch** [scriptPilot/app-framework#727]
Help please. I ran through docs 5 times and found that i need to build, then `npm run studio`. But it says me i should define playStoreId. What is it? Where should I get it? I am confused.

**Comment by outluch** [comment]
Ok.. I actually found. I looked at demo project. But it was very not obvious.

**Comment by fdbeirao** [comment]
For anyone that gets here like I did, the`relevant content of `config.json` in the demo project is:

```
  "appStoreId": "de.scriptpilot.app-framework",
  "playStoreId": "de.scriptpilot.appframework",
```

👍

**Comment by outluch** [comment]
Oh, thanks for that. I forgot haw I hate it when someone says in topic: I found solution, thank you all :D And no solution provided

**Comment by scriptPilot** [comment]
Yes, type your ID like that ... I will update the docs to me more clear in that point. Thanks @fdbeirao
